service/controller/resource/drainerfinalizer/error.go

```diff
@@ -1,11 +1,18 @@
 package drainerfinalizer
 
 import (
-	"strings"
+	"regexp"
 
 	"github.com/giantswarm/microerror"
 )
 
+var (
+	// noActiveLifeCycleActionRegExp is a fuzzy regular expression to match
+	// Autoscaling errors which we have to string match due to the lack of proper
+	// error types in the AWS SDK.
+	noActiveLifeCycleActionRegExp = regexp.MustCompile(`(?m)[nN][oO].*[lL][iI][fF][eE].*[cC][yY][cC][lL][eE].*[fF][oO][uU][nN][dD]`)
```

Comment:
This is the dumbest idea I could have come up with. Ideas welcome. It is just that the lower/upper case changed here again and we need some solution to fuzzy match that. Not a regexp hero here.

Reply:
You could add `i` flag instead. `(?m)` -> `(?im)`.

https://stackoverflow.com/questions/15326421/how-do-i-do-a-case-insensitive-regular-expression-in-go

Reply:
I'm also not sure why you need `m` flag.

Reply:
Nice one. That works.